DSharpPlus/Entities/Interaction/Application/DiscordApplicationCommandType.cs

```diff
@@ -22,7 +22,7 @@ public enum DiscordApplicationCommandType
     MessageContextMenu = 3,
 
     /// <summary>
-    /// Inbound only: An auto-complete option is being interacted with.
+    /// This command serves as the primary entry point into the app's activity.
     /// </summary>
-    AutoCompleteRequest = 4,
+    ActivityEntryPoint = 4,

```

Comment:
Announce this as a breaking change since the integer value is staying the same but the field name is not.

Reply:
anyone referencing this at all was already fundamentally broken, but we can have a note on discord